code/WorkInProgress/Electronics.dm

```diff
@@ -382,7 +382,7 @@
 ////////////////////////////////////////////////////////////////?
 /obj/item/electronics/soldering
 	name = "soldering iron"
-	icon = 'icons/obj/electronics.dmi'
+	icon = 'icons/obj/machinery/electronics.dmi'
```

Comment:
Just looking at one of the first things here, I disagree with any `/obj/item/foo` having a sprite in `icons/obj/item/machinery` because we have a strong coupling with `/obj/machinery` being a big part of the codebase. I know that their designation in code does not have to correspond to the "category" of sprite they use, but I would probably consider leaving anything that is using `icons/obj/electronics.dmi` out of this.